package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "ansi-colors": "4.1.1",
     "browser-stdout": "1.3.1",
-    "chokidar": "3.5.3",
+    "chokidar": "^3.6.0",
```

Comment:
Mocha is pinning to exact versions in manifest by convention. While I'd also suggest reconsidering that decision, changing that probably doesn't belong in this PR @simhnna?

```suggestion
    "chokidar": "^3.6.0",
```

Reply:
Sorry, I missed the context there!